downstairs/src/region.rs

```diff
@@ -753,12 +785,13 @@ impl Extent {
     fn create_copy_file(
         &self,
         mut copy_dir: PathBuf,
-        extension: Option<&str>,
+        extension: Option<ExtentExtension>,
```

Comment:
Alternatively you could have the "no extension" variant in `ExtentExtension` or maybe just use `FileType` directly.